withings_sync/withings2.py

```diff
@@ -177,26 +178,59 @@ def refresh_accesstoken(self):
                 " script again to obtain a new link."
             )
 
-        self.user_config["access_token"] = body.get("access_token")
-        self.user_config["refresh_token"] = body.get("refresh_token")
-        self.user_config["userid"] = body.get("userid")
+        self.user_config[self.api_user_id]["access_token"] = body.get("access_token")
+        self.user_config[self.api_user_id]["refresh_token"] = body.get("refresh_token")
+        self.user_config[self.api_user_id]["userid"] = body.get("userid")
 
 
 class WithingsAccount:
     """This class gets measurements from Withings"""
 
-    def __init__(self):
-        self.withings = WithingsOAuth2()
+    def __init__(self, api_user_id):
+        self.api_user_id = str(api_user_id)
+        self.withings = WithingsOAuth2(self.api_user_id)
+
+    def get_garmin_username(self):
+        """get Garmin username"""
+        if not self.withings.user_config[self.api_user_id].get("garmin_username"):
+            garmin_username = input("Garmin username : ")
```

Comment:
Question (for this and following input's): Can we use something like input with timemout?